tests/savepoint/test_translate.py

```diff
@@ -34,13 +34,24 @@ def success_array(computed_data, ref_data, eps, ignore_near_zero_errors, near_ze
         np.logical_and(np.isnan(computed_data), np.isnan(ref_data)),
         compare_arr(computed_data, ref_data) < eps,
     )
-    if ignore_near_zero_errors:
+    if isinstance(ignore_near_zero_errors, dict):
+        if ignore_near_zero_errors.keys():
+            near_zero = ignore_near_zero_errors["near_zero"]
+            success = np.logical_or(
+                success,
+                np.logical_and(
+                    np.abs(computed_data) < near_zero,
+                    np.abs(ref_data) < near_zero,
+                ),
+            )
+    elif ignore_near_zero_errors:
```

Comment:
I don't see any logic here involving comparing a variable name, which seems like it would be necessary?

Is our test code complicated enough that we need some unit tests for these helper functions?

Reply:
So the way it checks the variable is by first setting all fields to have ignore_near_zero_errors to False, then with the override, it reads the yaml variable and change it to True. I think we're probably okay here?

Reply:
Oh but you're right that we probably want to make sure that the specified variable is within the list of variables to be compared.

Reply:
the variable name comparison is already in the code, it was just set to Tru/False rather than a value